stdlib/public/Darwin/Accelerate/vForce_Operations.swift

```diff
@@ -0,0 +1,3435 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Accelerate
+
+// Array-Oriented Arithmetic and Auxiliary Functions
+
+extension vForce {
+    
+    // MARK: Ceiling
+    
+    /// Returns the ceiling of each element in `vector`, single-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Returns: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func ceil<U>(_ vector: U)  -> [Float]
+        where
+        U : _ContiguousCollection,
+        U.Element == Float {
+            
+            let result = Array<Float>(unsafeUninitializedCapacity: vector.count) {
+                buffer, initializedCount in
+                
+                ceil(vector,
+                     result: &buffer)
+                
+                initializedCount = vector.count
+            }
+            
+            return result
+    }
+    
+    /// Calculates the ceiling of each element in `vector`, writing the result to `result`, single-precision.
+    ///
+    /// - Parameter _ vector: Input values.
+    /// - Parameter result: Output values.
+    @inline(__always)
+    @available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *)
+    public static func ceil<U, V>(_ vector: U,
+                                  result: inout V)
+        where
+        U : _ContiguousCollection,
+        V : _MutableContiguousCollection,
+        U.Element == Float, V.Element == Float {
+            
+            precondition(vector.count == result.count)
+            
+            var n = Int32(vector.count)
```

Comment:
Can you add a `// TODO:` here to make this work for `count > Int32.max` at some point in the future? It doesn't need to be done now, but it's an obvious enhancement.

Reply:
That would require a change to the Accelerate.vForce API itself where `n` is an `Int32` (for example `[vvceilf](https://developer.apple.com/documentation/accelerate/1470395-vvceilf)`)

Reply:
We can handle this for users at the Swift wrapper layer if we want to, by simply calling `vvceilf` in a loop, processing 0x40000000 elements at a go. It's not a huge deal in the short term, because you can't actually allocate a contiguous block of Int32.max floats on any iOS platform (you *can* do it on macOS, however), which is why I'm perfectly content to wait to do this in the future.

Reply:
Todo added.